index.html

```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <link rel="stylesheet" href="/dist/tutor.css" />
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,300,300italic,400italic,900,700' rel='stylesheet' type='text/css'>
+    <link href='http://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
```

Comment:
I think it should go in a separate less file which defines 2 mixins (for both of the fonts). When are they "generally" used?